application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -65,21 +64,20 @@ public static String escapeMarkdown(String text) {
      * in Discord.
      *
      * @param guild the {@link Guild} that contains the command
-     * @param fullCommand the command's name with its optional subcommand
+     * @param commandPath the command's name with its optional subcommand
      * @return Formatted string for the mentioned slash command
      * @throws IllegalArgumentException when the command isn't found in the guild
      */
-    public static RestAction<String> mentionSlashCommand(Guild guild, @NotNull String fullCommand) {
-        List<String> fullCommandSplit = List.of(fullCommand.split(" "));
+    public static RestAction<String> mentionSlashCommand(Guild guild, String commandPath) {
+        String commandName = commandPath.split(" ", 1)[0];
```

Comment:
a bit fragile. what if there is no space? like when u do `mentionSlashCommand(guild, "ban")`. does it still work then or crash? it should be safe.

Reply:
Good thing you double checked on this. I did a bunch of tests in JShell and it turns out `split(" ", 1)` doesn't split anything. However, `split(" ", 2)[0]` will extract the first word of the command path properly. If the command is only one word, it works fine too. Considering this, what would you suggest to make it safer ? I can only think about a null check.

Reply:
null check is obsolete, since your method param is implicitly annotated with `@Nonnull`.

If that `[0]` is always safe, im fine with it

Reply:
> what if there is no space?

yes it'll work in that case

Reply:
it'll just return the word